modules/tracks/stores/FeedStore.js

```diff
@@ -0,0 +1,27 @@
+var toImmutable = require('nuclear-js').toImmutable;
+var Store = require('nuclear-js').Store;
+var RECEIVE_FEED = require('../actionTypes').RECEIVE_FEED;
+
+module.exports = new Store({
+    getInitialState: function () {
+        return toImmutable({ tracks: [] });
```

Comment:
`Uncaught TypeError: Cannot read property 'concat' of undefined`

From the `concat` call in `receiveFeed`.